soc/silabs/silabs_siwx91x/Kconfig

```
@@ -14,3 +14,19 @@ config SOC_SILABS_SLEEPTIMER
 	bool
 	help
 	  The Sleeptimer HAL module is used for SIWX91X.
+
+config SIWX91X_SILABS_COMMANDER_SIGN
+	bool "Silicon Labs Commander Signing Support"
+	depends on BUILD_OUTPUT_BIN
+	help
+		Activates the signing process for the Zephyr binary using Silicon Labs Commander during the build process.
```

Help text needs to be indented with 1 tab and 2 spaces, same for other options. Also try to keep a max line length of 100 chars.

```suggestion
	help
	  Activates the signing process for the Zephyr binary using Silicon Labs Commander
	  during the build process.
```

---

If all comments have been addressed, could I get approval?